README.md

```diff
@@ -1,2 +1,45 @@
+
 # Enumerables
-In this project we need to create enumerables (my_each, my_each_with_index , my_all, my_any, my_none,my_inject, my_map) using procs and yield
+
+> Enumerables project. In this project we need to create enumerables (my_each, my_each_with_index , my_all, my_any, my_none,my_inject, my_map) using procs and yield. In this project we learn alot about yield which is very helpfull in rails section.
+
+Additional description about the project and its features.
+
+## Built With
+
+- Ruby
+
+## Getting Started
+
+**Need web browser.** to download Repository
+**Need Terminal** to run the code
+
+To get a local copy up and running follow these simple example steps.
+
+### Download Repository (https://github.com/Gabkings/Enumerables)
+### open Terminal
+### cd enumerable
+### ruby enumerable.rb
```

Comment:
- [ ] **OPTIONAL** You can add the format to code by using the (`) backquote

It will `look like this` if you surround it with 1 backquote at the beginning and 1 backquote on the end
```
And it will look like this
If you surround it with 3 backquotes in the first line
and 3 backquotes in the last line
```
![image](https://user-images.githubusercontent.com/53256139/94956834-6b8ca000-04b2-11eb-95b2-5d2eaa7f6b63.png)